test/fw/ptl/utils/plugins/ptl_test_loader.py

```diff
@@ -191,7 +194,31 @@ def prepareTestLoader(self, loader):
 
         def check_loadTestsFromNames(names, module=None):
             rv = old_loadTestsFromNames(names, module)
-            self.check_unknown()
+            test_list = self.__tests_list_copy
```

Comment:
@shilpakodli As per our offline discussion, we don't need this change we just need to pass a new directory in `old_loadTestsFromNames` along with names. Also, you won't need a new argument to `set_data` after new changes